hnn_core/network.py

```diff
@@ -1497,6 +1525,49 @@ def update_weights(self, e_e=None, e_i=None,
         if copy:
             return net
 
+    def get_synaptic_gains(self):
+        """Retrieve gain values for different connection types in the network.
+
+        This function identifies excitatory and inhibitory cells in the network
+        and retrieves the gain value for each type of synaptic connection:
+        - excitatory to excitatory (e_e)
+        - excitatory to inhibitory (e_i)
+        - inhibitory to excitatory (i_e)
+        - inhibitory to inhibitory (i_i)
+
+        The gain is assumed to be uniform within each connection type, and only
+        the first connection's gain value is used for each type.
+
+        Returns
+        -------
+        dict: A dictionary with the connection types ('e_e', 'e_i', 'i_e',
```

Comment:
```suggestion
        values : dict
             A dictionary with the connection types ('e_e', 'e_i', 'i_e',
```

to be consistent with the docstring formatting standard